doit/task.py

```diff
@@ -230,6 +233,7 @@ def __init__(self, name, actions, file_dep=(), targets=(),
         self.values = {}
         self.verbosity = verbosity
         self.custom_title = title
+        self.cfg_defaults = cfg_defaults
```

Comment:
I think `cfg_values` would be a better name.

Reply:
Changed.